config/katello_defaults.yml

```diff
@@ -35,6 +35,9 @@ common:
   redhat_repository_url: https://cdn.redhat.com
   consumer_cert_rpm: 'katello-ca-consumer-latest.noarch.rpm'
 
+  content_action_accept_timeout: 20 #in seconds
+  content_action_finish_timeout: 3600 #in seconds
```

Comment:
What about making these admin configurable settings as opposed to ones that require server restart?